[DS-11] camera ~ facebook icons 추가

HeejunShon opened the pull request:
### what?
camera ~ facebook 아이콘 추가
### how?
resources/icons/icons/index.ts 에서 export 및 src/Icon.stories.tsx에서 import
### why?
아이콘 스토리북에 추가하기 위하여

HeejunShon committed:
chagned image name

HeejunShon committed:
changed image name

HeejunShon committed:
Merge branch 'master' of https://github.com/planzcoffee/planz-uikit

HeejunShon committed:
Merge branch 'master' of https://github.com/planzcoffee/planz-uikit

HeejunShon committed:
updated from upstream

HeejunShon committed:
v1.2.2

HeejunShon committed:
Merge branch 'master' of https://github.com/planzcoffee/planz-uikit

HeejunShon committed:
[DS-11] camera ~ facebook icons 추가

### what?
camera ~ facebook 아이콘 추가
### how?
resources/icons/icons/index.ts 에서 export 및 src/Icon.stories.tsx에서 import
### why?
아이콘 스토리북에 추가하기 위하여

planzcoffee commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "planz-uikit",
-  "version": "1.2.1",
+  "name": "planz-uikit-hj",
```
수정 부탁드립니다!

planzcoffee reviewed (commented):
package.json 수정부탁드립니다.  🙏

Jijuan commented on src/Icon/Icon.tsx:
```diff
@@ -0,0 +1,32 @@
+import { jsx } from '@emotion/core';
+
+// internal modules
+import * as icons from '@/icons/icons';
```
[6] import * as icons from '@/icons/icons' 부분이 개발할때는 편하고 보기도 좋긴한데, 항상 모든 icons 들을 가져와야 하는 구조같아요. 수정 부탁드립니다.

HeejunShon committed:
[DS-11] camera ~ facebook icons 추가 및 package.json 원복

### what?
camera ~ facebook 아이콘 추가, listOfIcons 추가 및 package.json name 복구
### how?
resources/icons/icons/index.ts 에서 export 및 src/Icon.stories.tsx에서 import
### why?
아이콘 스토리북에 추가하기 위하여

Jijuan reviewed (commented):
icons/icons 라는 파일구조가 다소 혼동을 초래할것으로 보입니다.

.DS_Store는 제거해주세요.

HeejunShon committed:
updated list of icons

HeejunShon committed:
[DS-11] camera ~ facebook icons 추가 및 package.json 원복

    ### what?
    camera ~ facebook 아이콘 추가, listOfIcons 추가 및 package.json name 복구
    ### how?
    resources/icons/icons/index.ts 에서 export 및 src/Icon.stories.tsx에서 import
    ### why?
    아이콘 스토리북에 추가하기 위하여
